packages/@aws-cdk/aws-codepipeline-api/lib/action.ts

```diff
@@ -153,6 +153,14 @@ export interface CommonActionConstructProps {
 export interface ActionProps extends CommonActionProps, CommonActionConstructProps {
   category: ActionCategory;
   provider: string;
+
+  /**
+   * The region this Action resides in.
+   *
+   * @default the Action resides in the same region as the Pipeline
```

Comment:
Don't we want all the details about cross region replication buckets here as well?

Reply:
Added.